src/Dialect/ONNX/ONNXOps.cpp

```diff
@@ -384,6 +384,8 @@ static LogicalResult processConvPadParam(T *op, ArrayRef<int64_t> inputShape,
       // -1)*dilation + 1".
       auto sumOfPad = (outputSize - 1) * strideVal +
                       ((kernelSize - 1) * dilationVal + 1) - inputSize;
+      if (sumOfPad < 0)
+        sumOfPad = 0;
```

Comment:
Just wonder if this would be an assertion? Is it possible that `sumOfPad < 0`?

Reply:
I've added a comment to explain the case where this happens. 

It can happen in the case when the dimension of the filter size is 1.

example: 

`outputSize = 10, strideVal = 2, kernelSize = 1, dilationVal = 1, inputSize = 20`

then 
```
sumOfPad = (outputSize - 1) * strideVal + ((kernelSize - 1) * dilationVal + 1) - inputSize
                      = (10 - 1) * 2 + ((1 - 1) * 1 + 1) - 20
                      = 18 + 1 - 20
                      = - 1